test/functionalTest/cases/2596_retrieve_entity_without_attrs/retrieve_entity_without_attrs.test

```diff
@@ -32,7 +32,7 @@ brokerStart CB
 #
```

Comment:
Rollback de change in this file done in PR https://github.com/telefonicaid/fiware-orion/pull/3751

NTC (informative)